### PR TITLE
Task/some tidy up

### PR DESCRIPTION
**Pull request by talor-hammond**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "editor.codeActionsOnSave": {
```
Comment (talor-hammond) [inline .vscode/settings.json:2]: this sets eslint up to automatically format files when you save

File: package.json [hunk 2]
```diff
@@ -9,6 +9,11 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.1",
+    "@types/jest": "^26.0.20",
```
Comment (talor-hammond) [inline package.json:12]: these are just dependencies that you need for typescript.

you can optionally use it if you want to play around (you can still use .js files no sweat)

File: package.json [hunk 3]
```diff
@@ -9,6 +9,11 @@
...
+    "@types/node": "^14.14.33",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.2",
+    "node-sass": "^5.0.0",
```
Comment (talor-hammond) [inline package.json:16]: `node-sass` lets you use .scss files now

File: src/components/pages/AboutMe.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown"
+import "./AboutMe.scss"
+
+interface Props {
```
Comment (talor-hammond) [inline src/components/pages/AboutMe.tsx:5]: heres an example of TypeScript react-component

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (talor-hammond) [inline tsconfig.json:1]: `tsconfig.json` lets you define the rules you want for your TypeScript setup
can leave this config and update if you ever need to

File: .vscode/settings.json (hunk 1 above)
Comment (talor-hammond) [reply .vscode/settings.json:2]: just remember to install the eslint extension for vscode otherwise it no work

**Review by DeonLeer — APPROVED** [review]

**Pull request merged**